monasca-alarms/monasca_alarm_definition.py

```diff
@@ -564,26 +555,34 @@ def main(args=None):
                         " or a token via --os-auth-token or"
                         " env[OS_AUTH_TOKEN]")
 
-    kwargs = {
+    all_keystone_kwargs = {
+        'auth_url': args.os_auth_url,
         'username': args.os_username,
         'password': args.os_password,
-        'keystone_token': args.os_auth_token,
-        'auth_url': args.os_auth_url,
-        'service_type': args.os_service_type,
-        'endpoint_type': args.os_endpoint_type,
-        'os_cacert': args.os_cacert,
+        'tenant_name': args.os_tenant_name,
         'project_id': args.os_project_id,
         'project_name': args.os_project_name,
+        'project_domain_name': args.os_project_domain_name,
+        'user_domain_name': args.os_user_domain_name,
         'domain_id': args.os_domain_id,
-        'domain_name': args.os_domain_name,
+        'domain_name': args.os_domain_name
+    }
+    ks_kwargs = dict((k, v) for k, v in all_keystone_kwargs.iteritems() if v)
+
+    kwargs = {
+        'keystone_kwargs': ks_kwargs,
+        'keystone_token': args.os_auth_token,
+        'endpoint_type': args.os_endpoint_type,
+        'os_cacert': args.os_cacert,
+        'service_type': args.os_service_type,
         'insecure': args.insecure,
         'monasca_api_url': args.monasca_api_url,
         'api_version': args.monasca_api_version,
         'verbose': args.verbose
     }
 
     if not monascaclient_found:
-        print("python-monascaclient>=1.6.0<1.7.0 is required", file=sys.stderr)
+        print("python-monascaclient>=1.6.0 is required", file=sys.stderr)
```

Comment:
I think we should change that now into `>=1.7.0`. Or even actually remove that to be honest. `monasca-alarm` image is based on top of `monasca-client` image meaning that client will be always available.

Reply:
done